### DISCUSSION TITLE
Use the "index" word in the path

### DISCUSSION THREAD

**Discussion by heidarv** [nuxt/nuxt#20731] [Questions] [answered]
I need to have the word "`index`" in the path like `/library/index`. When I create an `index` directory with an `index.vue` file in it under the `library` directory the path doesn't work and responds "page not found".

When I use `router.options.ts` like:
```
  path: "/library/index",
  component: () =>
  import("~/pages/library/index/index.vue"),

```
this will work only the first time page loads at the address and not when routed from client side.

**Comment by warflash** [comment, marked as answer]
You could use the alias property `definePageMeta` provides, I've put up an example real quick -> https://stackblitz.com/edit/github-jh62hm-6n7ejr?file=pages%2Findex.vue,pages%2Flibrary%2Findex.vue

**Reply by heidarv** [reply]
That was great. Thank you.
